disnake/webhook/interaction.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import List, Optional
+
+from disnake.abc import Snowflake
+from disnake.embeds import Embed
+from disnake.file import File
+from disnake.flags import MessageFlags
+from disnake.mentions import AllowedMentions
+from disnake.ui.action_row import Components, MessageUIComponent
+from disnake.ui.view import View
+from disnake.utils import MISSING
+from disnake.webhook.async_ import Webhook, WebhookMessage
```

Comment:
These could probably be moved into a `TYPE_CHECKING` block

Reply:
886028058deb5f9ec2704b028b54298e975f0be6